### PR TITLE
Issue 2 view the entries

### PR DESCRIPTION
**Pull request by JimmyTran1206**
Animation

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -22,13 +22,14 @@
       <div class="row">
         <div class="column-full">
           <h1 class="journal-header">Code Journal</h1>
-          <a href="#">
+          <a class="entries-link" href="#">
             <h1 class="entries">Entries</h1>
```
Comment (shawnkost) [inline index.html]: You can only have 1 `h1` element on a page, or else it gets flagged as an accessibility problem. I would just put the text directly inside the `a` tag, or use a different element like `span` inside the `a`

File: js/main.js [hunk 2]
```diff
@@ -23,5 +25,99 @@ function formSave(event) {
...
+  const $pEntryNotes = document.createElement('p');
+  $pEntryNotes.className = 'entry-notes';
+  $pEntryNotes.innerText = entry.notes;
+  $divColumnHalf2.append($pEntryNotes);
+
+  return $liEntry;
+}
+
+const $ulEntryList = document.querySelector('ul.entry-list');
+function DOMContentLoadedHandler(event) {
```
Comment (shawnkost) [inline js/main.js]: Function names should be lowercase. The only time we'll capitalize is when working in React, or when creating a `Class` in JavaScript

**Review by shawnkost — CHANGES_REQUESTED** [review]
Looking good Jimmy! Just a couple tweaks here

File: js/main.js (hunk 2 above)
Comment (JimmyTran1206) [reply js/main.js]: Thank you Shawn, I have changed the function name:
image

File: index.html (hunk 1 above)
Comment (JimmyTran1206) [reply index.html]: Thank you for your reminder, Shawn. I have changed the h1-tag to span-tag
image

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**